geoengine/workflow.py

```diff
@@ -520,11 +521,17 @@ def save_as_dataset(
         session = get_session()
 
         request_body = {
-            'name': name,
+            'displayName': display_name,
             'description': description,
             'query': query_rectangle,
         }
 
+        if name is not None:
```

Comment:
do you really need that if statement? Why can't you just set it to None in the request body?

Reply:
well it will cause the tests to fail and i have no idea how to fix that. Also it does not hurt to do it this way. We had an issue with that (can't remember) where the empty parameter is send and something breaks.

Reply:
oh because the test expects the attribute to be absent instead of null? Can't you adjust the test, to expect it to be null?

Reply:
i  can't figure out how to adjust the test to not fail, sorry :(

Reply:
[0f55085](https://github.com/geo-engine/geoengine-python/pull/141/commits/0f5508517a410f7dc3c9b4a01dc1e52516b657ae)